tutorials/frontend/deploy_ssd_gluoncv.py

```diff
@@ -29,14 +30,16 @@
 #
 # .. note::
 #
-#   Currently we support compiling SSD on CPU only.
-#   GPU support is in progress.
+#   We support compiling SSD on bot CPUs and GPUs now.
 #
 #   To get best inference performance on CPU, change
 #   target argument according to your device and
 #   follow the :ref:`tune_relay_x86` to tune x86 CPU and
 #   :ref:`tune_relay_arm` for arm cpu.
 #
+#   To get best performance fo SSD on intel graphics,
+#   change target argument to 'opecl -device=intel_graphics'
```

Comment:
typo in 'opecl'
Anyway it is better to use `target = tvm.target.intel_graphics()`